test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/MiscCryptoSuite.java

```diff
@@ -67,6 +70,42 @@ public static void main(String... args) {
 		);
 	}
 
+	private HapiApiSpec sysAccountKeyUpdateBySpecialWontNeedNewKeyTxnSign(){
+		String sysAccount = "0.0.977";
+		String randomAccountA = "randomAccountA";
+		String randomAccountB = "randomAccountB";
+		String firstKey = "firstKey";
+		String secondKey = "secondKey";
+
+		return defaultHapiSpec("sysAccountKeyUpdateBySpecialWontNeedNewKeyTxnSign")
+				.given(
+						withOpContext((spec, opLog) -> {
+							spec.registry().saveKey(sysAccount, spec.registry().getKey(GENESIS));
+						}),
+						newKeyNamed(firstKey).shape(SIMPLE),
+						newKeyNamed(secondKey).shape(SIMPLE)
+				)
+				.when(
+						cryptoCreate(randomAccountA)
+								.key(firstKey),
+						cryptoCreate(randomAccountB)
+								.key(firstKey)
+								.balance(ONE_HUNDRED_HBARS)
+				)
+				.then(
+						cryptoUpdate(sysAccount)
+								.key(secondKey)
```

Comment:
I think here we should add a `.signedBy(GENESIS)` to override the default `HapiCryptoUpdate` behavior to sign with the new key (here, `secondKey`).

Otherwise we aren't testing the new behavior enabled by this PR.

(When we call `HapiSpecOperation.signedBy(String... )`, we ensure the listed keys are the **only** signers.)